app/src/main/kotlin/com/simplemobiletools/musicplayer/databases/SongsDatabase.kt

```diff
@@ -196,7 +196,7 @@ abstract class SongsDatabase : RoomDatabase() {
                     execSQL("ALTER TABLE tracks ADD COLUMN genre TEXT NOT NULL DEFAULT ''")
                     execSQL("ALTER TABLE tracks ADD COLUMN genre_id INTEGER NOT NULL DEFAULT 0")
 
-                    execSQL("CREATE TABLE `genres` (`id` INTEGER NOT NULL PRIMARY KEY, `title` TEXT NOT NULL, `track_cnt` INTEGER NOT NULL)")
+                    execSQL("CREATE TABLE `genres` (`id` INTEGER NOT NULL PRIMARY KEY, `title` TEXT NOT NULL, `track_cnt` INTEGER NOT NULL, `album_art TEXT NOT NULL`)")
```

Comment:
guess the quote should end after album_art, not null, like
`album_art` TEXT NOT NULL
instead of 
`album_art TEXT NOT NULL`

Reply:
right, fixed it